index.bs

```diff
@@ -361,7 +361,70 @@ Report long tasks {#report-long-tasks}
 Security & privacy considerations {#priv-sec}
 ===============================================
 
-Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about the source of the long task.
-There is a 50ms threshold for long tasks. Together this provides adequate protection against security attacks against browser.
+Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about
+the source of the long task. There is a 50ms threshold for long tasks. Together this provides adequate
+protection against security attacks against the browser itself.
```

Comment:
What do you mean by "against the browser itself"? Is this protecting against browser runtime vulnerabilities? cross-origin leaks? Something else?

Reply:
This was copy-paste from existing, and I don't understand either so replacing with cross-origin leak.